src/Common/src/TypeSystem/Common/TypeSystemException.cs

```diff
@@ -193,6 +193,10 @@ public static string OwningModule(MethodDesc method)
 
             public static string OwningModule(TypeDesc type)
             {
+#if TYPE_LOADER_IMPLEMENTATION
```

Comment:
(I thought @davidwrighton wanted to implement name and namespace on nometadata types with poisoned strings so that we don't accidentally rely on the presence of metadata for nometadata types in unsuspected places. Poisoned strings cannot be used to format user visible names)